bf2fc6cc711aee1a0c2a/kas-fleet-manager
# making kind & items mandatory for service account list

## Description

Closes: https://issues.redhat.com/browse/MGDSTRM-2842

Updated the openapi spec to make kind & items mandatory for list service accounts

## Verification Steps

Remove all the service accounts from the org
Call list end-point, 
http://localhost:8000/api/managed-services-api/v1/serviceaccounts

the response will look like

expected:
``` { "kind":"ServiceAccountList" "items":[], }```

## Type of change


## Checklist (Definition of Done)

- [ x] All acceptance criteria specified in JIRA have been completed
